subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/BuildVerdict.kt

```diff
@@ -0,0 +1,18 @@
+package com.avito.android.build_verdict
+
+internal data class GsonableError(
```

Comment:
why `gson` in naming? i see nothing about concrete json serialization library in it's signature

Reply:
Throwable can be serialize to the JSON. I see two options to fix:
- Create a wrapper object
- Register adapter
I prefer a wrapper because it is closer to the domain and I want to decide what data I will extract from Throwable

Reply:
yes i know that you need custom serialization; i prefer adapter here even if i don't like it's API, trigger for me is naming, if i can't justify name - something is wrong with abstraction 
so what about specific library in data class name? (why? nothing about gson here, it could be Jacksonable or Moshiable), leaked detail in naming

Reply:
Jsonable

Reply:
rename to `Error` without adapters